Test/WebTestCase.php

```diff
@@ -341,7 +342,7 @@ private function loadFixtureClass(Loader $loader, $className)
      *
      * @return UploadedFile
      */
-    private function getFile($file, $data, $ext, $mime)
+    protected function getFile($file, $data, $ext, $mime)
```

Comment:
This bundle should be more generic as possible.
Anyone should be able to generate any kind of file, and you cannot foresee a method for every possible type of file.